ansible/host_vars/geth-01.ih-eu-mda1.nimbus.holesky.yml

```diff
@@ -17,6 +17,11 @@ redirect_ports:
     dst: '{{ test_api_beacon_node_rest_port }}'
     comment: 'Test Beacon API ({{ test_api_beacon_node_branch }})'
 
+# Open Nginx Ports
+open_ports_default_comment: 'Nginx'
+open_ports_default_chain: 'SERVICES'
+open_ports_list: [ { port: 80 }, { port: 443 } ]
```

Comment:
You don't need this, you're overriding the values set here this way, which is bad:
https://github.com/status-im/infra-nimbus/blob/0c26a3a8f1fc20ae25378fa4d2153e172173078b/ansible/group_vars/nimbus.holesky.yml#L156-L163

Reply:
Sorry, Thanks for explaining.